packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-vm-console-certificates/tasks/main.yml

```diff
@@ -54,14 +54,18 @@
     copy:
       content: "{{ ovirt_ca_key }}"
       dest: "{{ ovirt_vmconsole_ca_file }}"
+      owner: 'ovirt-vmconsole'
+      group: 'ovirt-vmconsole'
       mode: 0644
```

Comment:
This is a key, so shouldn't we change to 640 or even 600?

Reply:
it's actually not, it's the /etc/pki/ovirt-vmconsole/ca.pub
used in sshd_config as
TrustedUserCAKeys /etc/pki/ovirt-vmconsole/ca.pub

Reply:
Ah, so it's completely incorrect name for this variable, it should be named `ovirt_ca_cert_ssh`, because it's oVirt CA public certificate formatted using OpenSSH cert format:

https://github.com/oVirt/ovirt-engine/blob/master/backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/HostEnrollCertificateInternalCommand.java#L66

Reply:
But the mode it OK then and it should be readable, so why changing the owner?